src/iuse.h

```diff
@@ -232,7 +233,9 @@ class iuse_actor {
     virtual ~iuse_actor() { }
     virtual void load( JsonObject &jo ) = 0;
     virtual long use( player &, item &, bool, const tripoint& ) const = 0;
-    virtual bool can_use( const player &, const item &, bool, const tripoint& ) const { return true; }
+    virtual ret_val<bool> can_use( const player &, const item &, bool, const tripoint& ) const {
```

Comment:
Why not move this implementation into the cpp and forward declare `ret_val` and move the include of "ret_val.h" into the cpp file as well.